draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -350,9 +350,25 @@ control algorithms have the opportunity to improve the state of the art.
 
 ### Fairness within the Alternate Congestion Control Algorithm.
 
-When multiple competing flows all using the same
+When multiple competing flows all use the same
 alternate congestion control algorithm, the proposal should
 explore how the capacity is shared among the competing flows.
+Capacity fairness can be important when a small number of similar
+flows compete to fill a bottleneck. It can however also not be useful:
+for example when comparing flows seek to send at different rates or
+when some of the flows do not last sufficiently long to approach
+asymptotic behavior.
+
```

Comment:
I am not sure that we shall mandate capacity fairness, even there. It kinds of dilute the "harm and starve" message.

Also, the state of the art is murky, because few deployments can actually ensure fair sharing, especially if different flows are experiencing different RTT. Today, we have:

* Reno: flows get shares inversely proportional to their RTT.
* Cubic: lots of efforts to ensure equal shares independently of RTT, but I have no particular experience with how good they do.
* BBR: flows get share. If you wait 10 seconds, competitors will learn the actual RTT, and sharing will be a bit more fair.

Getting better shares for small RTT may be unavoidable. Hosts with short RTT will track the state of the path more closely and react faster than hosts with long RTT. It may also be what the network wants: local flows get more bandwidth; their faster reaction implies a better network utilization.

Reply:
I agree on the comment about lower RTTs - i.e. shorter control loop.

Reply:
Placed this in a separate section for now, so a reader won't muddle  the two things.